contextproject-ms3/src/main/java/contextproject/controllers/PlaylistsController.java

```diff
@@ -0,0 +1,75 @@
+package contextproject.controllers;
+
+import contextproject.models.Playlist;
+import contextproject.models.Property;
+
+import java.util.ArrayList;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.event.EventHandler;
+import javafx.fxml.FXML;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+import javafx.scene.input.MouseEvent;
+
+public class PlaylistsController {
```

Comment:
I think we should rename some stuff, as it gets confusing by introducing PlaylistsController together with LibraryController. 

In my opinion it seems more logical to have a LibraryController (which is the playlist selector) and then a PlaylistController that controls a single playlist. The models are setup this way as well. :)